scores.js

```diff
@@ -28,9 +28,18 @@ avg /= betterScore.length;
 // round the average
 avg = Math.round(avg);
 
+/* ALTERNATIVE
+// compute the average
+avg = betterScore.reduce((x,y) => x+y);
+avg /= betterScore.length;
+
+// round the average
+avg |= 0;
```

Comment:
I think this is a loss of readability, why use a "language quirk" when a perfectly valid and readable alternative exists?
Also, I think this truncates the number, rather than rounding to the closest integer.